rust-app/src/lib.rs

```diff
@@ -2,9 +2,10 @@
 #![allow(incomplete_features)]
 #![feature(stmt_expr_attributes)]
 #![feature(adt_const_params)]
-#![feature(str_internals)]
 #![feature(type_alias_impl_trait)]
-#![feature(const_mut_refs)]
+#![cfg_attr(not(version("1.83")), feature(const_mut_refs))]
+// Hack to fix build with nightly-2024-11-22
```

Comment:
Did you somehow verify we're actually having an issue without this? Looks like an old nightly version already

Reply:
I think it's not an issue just with nightly anymore, but also with stable now

Reply:
Removed the comment